selfdrive/ui/qt/onroad.cc

```diff
@@ -458,24 +458,25 @@ void AnnotatedCameraWidget::drawLaneLines(QPainter &painter, const UIState *s) {
   }
 
   // paint path
-  QLinearGradient bg(0, height(), 0, height() / 4);
+  int max_y = std::max_element(scene.track_vertices.begin(), scene.track_vertices.end(),
+                               [](const QPointF &a, const QPointF &b) { return a.y() > b.y(); })->y();
+  QLinearGradient bg(0, height(), 0, max_y);
```

Comment:
There might be a smarter way of doing this... I tried using [CoordinateModes](https://doc.qt.io/qt-5/qgradient.html#CoordinateMode-enum) but couldn't figure out how to have [0.0, 1.0] not map to the bottom left/top right corners - we want it to be vertical gradient only